src/main/kotlin/lotto/model/NumberGenerator.kt

```diff
@@ -1,10 +1,8 @@
 package lotto.model
 
-import kotlin.collections.shuffled
-
 class NumberGenerator {
 
-    fun generateNumbers() = (MINIMUM_VALUE..MAXIMUM_VALUE).shuffled().take(6).sorted()
+    fun generateRandomNumbers(): List<Int> = (MINIMUM_VALUE..MAXIMUM_VALUE).shuffled().take(6).sorted()
```

Comment:
In TicketSeller, instead of directly depending on NumberGenerator, try creating a new interface and depend on that.
Then, make NumberGenerator an implementation of the new interface.

This way, TicketSeller can be tested not only for the count but also for specific lotto numbers.
If you have time, it would be great to give it a try! 😊